rules/always-return.js

```diff
@@ -139,6 +169,14 @@ module.exports = {
           ignoreLastCallback: {
             type: 'boolean',
           },
+          ignoreAssignmentVariable: {
+            type: 'array',
+            items: {
+              type: 'string',
+              pattern: '^[\\w]+$',
```

Comment:
nit(non-blocking): The char group is not needed here
```suggestion
              pattern: '^\\w+$',
```